.github/workflows/build-op-image.yaml

```diff
@@ -18,41 +19,118 @@ on:
       - "pkg/fluentd/utils/**"
       - "Makefile"
 
+  pull_request:
+    branches:
+      - "master"
```

Comment:
Similar here , maybe we should add "paths:" here too to avoid build images for non-code changes